### DISCUSSION TITLE
Someone remaking save system?

### DISCUSSION THREAD

**Discussion by UkrainianBanderasCat** [UkrainianBanderasCat/Digital-Logic-Sim2#58] [General]
Does someone remaking save system or I need to remake it?

**Comment by firecerne** [comment]
Maybe the problem comes from the new type of signal replacing the others.
Reverting the changes done for the side-independent signal system may work, which could let us build this version, it's been a long time since the last release.

**Reply by AlphaCodingPilot** [reply]
I also don't think being able to choose the signal type is necessary and could even lead to confusion.
